docs/rules/no-constant-condition.md

```diff
@@ -41,6 +41,15 @@ do{
 var result = 0 ? a : b;
 ```
 
+```js
+/*eslint no-constant-condition: "error"*/
+
+if(typeof x){
+  doSomething();
```

Comment:
Weird, I thought we had lint rules for that. I'll try to check today to see if I missed something in my analysis of style guide vs lint rules.